src/vendor/pdfjs/web/viewer.html

```diff
@@ -14,33 +14,36 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 
-Adobe CMap resources are covered by their own copyright and license:
-http://sourceforge.net/adobe/cmap/wiki/License/
+Adobe CMap resources are covered by their own copyright but the same license:
+
+    Copyright 1990-2015 Adobe Systems Incorporated.
+
+See https://github.com/adobe-type-tools/cmap-resources
 -->
 <html dir="ltr" mozdisallowselectionprint moznomarginboxes>
   <head>
+<script async defer src="/public/embed.js"></script>
```

Comment:
I reckon we probably only want/need `async` here, as that gives us some flexibility about when we instantiate what. (It's also what we recommended in [our own docs](https://h.readthedocs.io/en/latest/publishers/embedding/).)